composer.json

```diff
@@ -31,8 +31,6 @@
         "cakephp/utility": "Provides CakePHP security methods. Required for the JWT adapter and Legacy password hasher."
     },
     "license": "MIT",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
```

Comment:
we can probably keep prefer stable right?

Reply:
We can but it would be redundant without `"minimum-stability"` set to `dev` or `beta`

Reply:
Jep, but we have one less to always add and remove for each major.
Meh/^^